src/gnosissafe/GnosisSafeWrapper.sol

```diff
@@ -5,32 +5,35 @@ pragma solidity ^0.8.19;
 import { IGnosisSafe } from "./interfaces/IGnosisSafe.sol";
 
 import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 
 import { Enum } from "./lib/Enum.sol";
 import { BaseWrapper, IERC7399, IERC20 } from "../BaseWrapper.sol";
 
 /// @dev Safe Gnosis Flash Lender that uses individual Gnosis Safe contracts as source of liquidity.
-contract GnosisSafeWrapper is BaseWrapper, AccessControl {
+contract GnosisSafeWrapper is BaseWrapper, AccessControl, Initializable {
     error UnsupportedAsset(address asset);
     error FailedTransfer(address asset, uint256 amount);
     error InsufficientRepayment(address asset, uint256 amount);
 
     event LendingDataSet(address asset, uint248 fee, bool enabled);
+    event SafeSet(IGnosisSafe safe);
```

Comment:
indexed for asset & safe

Reply:
Does it make sense here to index `safe`? It will be set only once in each wrapper, during construction.

Reply:
It could be used as a reverse lookup I guess, it's pretty much the same in gas terms, so I'd do it, I've learnt the bad way that you should put as much data in the events as possible, even if it looks redundant or useless at the time